campusdiscussbackend/settings.py

```diff
@@ -43,6 +43,7 @@ def get_secret(setting, secrets=secrets):
 # Application definition
 
 INSTALLED_APPS = [
+    'streams.apps.StreamsConfig'
```

Comment:
You could add this line as the last item item of this list to maintain the order in which apps are added. Also you should put a comma to separate this from other items of the list. You should check for errors and wether the app has been added or not in the django admin page. 
To access it run `python manage.py runserver`, and check wether the app is added at `localhost:port(8000)/admin`.